strictEquals.js

```diff
@@ -0,0 +1,4 @@
+const strictEquals = (numOne, numTwo) => {
+  return (typeof numOne == typeof numTwo && numOne == numTwo)
+}
```

Comment:
Nice clean solution. Remember that since this is a one line return function you can use fat arrow's implicit return as well.

```JavaScript
const strictEquals = (numOne, numTwo) => typeof numOne == typeof numTwo && numOne == numTwo
```